src/commands/run.js

```diff
@@ -62,6 +62,10 @@ class RunCommand extends ApifyCommand {
         // NOTE: User can overwrite env vars
         const env = Object.assign(localEnvVars, process.env);
 
+        if (!userId) {
+            warning('You are running actor without your Apify account. Some feature like Apify proxy won\'t work. Call "apify login" to fix that.');
```

Comment:
You're not logged in with your Apify Account. Some features like Apify Proxy will not work. Call ...